src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -324,12 +324,17 @@ public Authentication authenticate(Authentication authentication) throws Authent
                     }
                 },
                 new UserDetailsService() {
-					
+
 					@Override
 					public UserDetails loadUserByUsername(String username) throws UsernameNotFoundException, DataAccessException {
 						// Retrieve the OicUserProperty to get the list of groups that has to be set in the OicUserDetails object.
 						LOGGER.fine("loadUserByUsername in createSecurityComponents called, username: " + username);
-						User u = User.get(username);
+						User u = User.get(username, false, Collections.emptyMap());
+						if (u == null) {
+							LOGGER.fine("loadUserByUsername in createSecurityComponents called, no user '" + username + "' found");
+							throw new UsernameNotFoundException(username);
+						}
+						LOGGER.fine("loadUserByUsername in createSecurityComponents called, user: " + u);
```

Comment:
This line is same with the next line.

Reply:
Do you mean line 327? That's trailing white space, deleted by my editor... You want me to restore the empty line?

Reply:
line 337 is same with line 338.

Reply:
Oops you're right - let me amend that. Something went wrong with my patch

Reply:
Done; removed the extra whitespace trimming too.